Makefile

```diff
@@ -136,11 +136,16 @@ build:
 
 .PHONY: operator-image
 operator-image: generate
-	$(CONTAINER_RUNTIME) build -f build/Dockerfile . -t $(OPERATOR_IMG)
+	$(CONTAINER_RUNTIME) build -f build/Dockerfile --no-cache --platform linux/amd64 -t $(OPERATOR_IMG)-amd64 --build-arg TAGRTEARCH=amd64 .
+	$(CONTAINER_RUNTIME) build -f build/Dockerfile --no-cache --platform linux/ppc64le -t $(OPERATOR_IMG)-ppc64le --build-arg TARGETARCH=ppc64le .
+	$(CONTAINER_RUNTIME) push ${OPERATOR_IMG}-amd64
+	$(CONTAINER_RUNTIME) push ${OPERATOR_IMG}-ppc64le
```

Comment:
Same question here as for the bundle image. Do we have to push images in the build step? If not strictly necessary I'd prefer to keep build and push separate.